docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
```

Comment:
If I understand correctly this means it is yet another implementation of the libmagic bindings, the ideal bindings would need to: 
1. Work with the current code(though could be that adapting it to different binding doesn't require a lot )
2. Have maintained packaging for CentOS/Fedora and Debian.
3. Available on pip.
So the only missing fact is whether ``file-magic`` has maintained packaging. For ``python-magic``, I think the only thing missing is pip. 
@gbenhaim - what you used on debian is the same ``python-magic``?